docs/conf/portland/2021/writing-day.rst

```diff
@@ -49,6 +49,28 @@ Write the Docs Meetups
 
 Organizing local Write the Docs meetup communities is a rewarding way to participate. During Writing Day, we'll have a table where we can share tips and best practices, especially in this time where all of our meetups are virtual.
 
+GitLab - Learn how to contribute to open-source docs
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+`GitLab <https://about.gitlab.com>`_ is an open core single application for
```

Comment:
I wasn't familiar with "open core" as a term and had to look it up. You might link out to a definition just in case.